src/pages/HomeContent.jsx

```diff
@@ -516,11 +430,11 @@ export default function HomeContent() {
             <Text
               style={{ color: "#80edff" }}
               onClick={() => {
-                setStep2(!allOpen)
-                setStep3(!allOpen)
+                setStep2(!allOpen);
```

Comment:
Can we just embed the tutorial instead of linking off the page? (it's a few lines up)

Reply:
I'll make a ticket. Resolving..